src/snowflake/snowpark/modin/plugin/_internal/where_utils.py

```diff
@@ -16,17 +16,13 @@ def validate_expected_boolean_data_columns(frame: InternalFrame) -> None:
     Returns:
         None
     """
-    frame_snowflake_identifier_to_data_type_map = (
-        frame.quoted_identifier_to_snowflake_type()
-    )
-
     if not all(
         isinstance(
-            frame_snowflake_identifier_to_data_type_map.get(
-                snowflake_quoted_identifier
-            ),
+            type,
```

Comment:
nit: should use `each_type` or similar since `type` is a special variable in python